mne/viz/tests/test_evoked.py

```diff
@@ -92,6 +92,8 @@ def test_plot_evoked():
         # plot with bad channels excluded
         evoked.plot(exclude='bads')
         evoked.plot(exclude=evoked.info['bads'])  # does the same thing
+        # spatial colors test
+        evoked.plot(spatial_colors=True)
```

Comment:
maybe just put spatial_colors=True in one of the 2 plots above. it avoids test time.